History.md

```diff
@@ -4,6 +4,7 @@
   * Add pumactl `thread-backtraces` command to print thread backtraces (#2053)
   * Configuration: `environment` is read from `RAILS_ENV`, if `RACK_ENV` can't be found (#2022)
   * `Puma.stats` now returns a Hash instead of a JSON string (#2086)
+  * `GC.comapct` is called before fork if available (#2093)
```

Comment:
@schneems small typo here, should be `GC.compact`

Reply:
Thanks!